src/components/ChatAIPage.tsx

```
@@ -0,0 +1,234 @@
+        if (!input.trim() || !tokenuser) return;
+
+        const userMessage: Message = { content: input, role: 'user' };
+        setMessages((prev) => [...prev, userMessage]);
+        setInput('');
+        setIsLoading(true);
+        setError(null);
+
+        try {
+            const response = await chatAIFeedback(input, tokenuser);
+            console.log('check', response);
+            if (response && response.metadata) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)